diambra/arena/stable_baselines/sb_utils.py

```diff
@@ -1,72 +1,48 @@
 from stable_baselines.common.callbacks import BaseCallback
 import cv2
-import os
-import time
-import json
 import numpy as np
 from pathlib import Path
 
 # Visualize Obs content
+def show_obs(observation, ram_states_list, n_actions, n_actions_stack, char_list, viz=False, wait_key=1):
```

Comment:
Nothing for now but we should consider adding `__repr__` methods to all these classes so we can just print them directly without conversion in the caller like here

Reply:
Yeah we discussed this in the past and I actually looked at it in this refactoring, but I am not sure if this is easily feasible, let me explain you why. The thing we are plotting here is the observation, that needs to be of type `gym.spaces`, this is a strict requirement for standardization and compatibility with RL libraries for example. I am not sure that extending that class with an additional method implementing the visualization would keep the compatiblity viable. 

You see what I mean? What do you think?

Reply:
Yeah good question. I would assume extending gym.spaces should be fine since whatever consumes that type would require a type derived from gym.spaces which would be the case. Might be worth trying, but again no need to do that now

Reply:
Sure, I created a dedicated issue for that: https://github.com/orgs/diambra/projects/4/views/4?pane=issue&itemId=38502627

Will look into that in the near future